sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java

```diff
@@ -603,6 +625,36 @@ private boolean getSuccess(SpanData span) {
 
     @Nullable
     public static String getHttpUrlFromServerSpan(Attributes attributes) {
+        String httpUrl = getHttpUrlFromServerSpanStableSemconv(attributes);
+        if (httpUrl != null) {
+            return httpUrl;
+        }
+        return getHttpUrlFromServerSpanOldSemconv(attributes);
+    }
+
+    @Nullable
+    private static String getHttpUrlFromServerSpanStableSemconv(Attributes attributes) {
+        String scheme = attributes.get(SemanticAttributes.URL_SCHEME);
+        if (scheme == null) {
+            return null;
+        }
+        String path = attributes.get(SemanticAttributes.URL_PATH);
+        if (path == null) {
+            return null;
+        }
+        String host = attributes.get(SemanticAttributes.SERVER_ADDRESS);
+        if (host == null) {
+            return null;
+        }
+        Long port = attributes.get(SemanticAttributes.SERVER_PORT);
+        if (port != null && port > 0) {
+            return scheme + "://" + host + ":" + port + path;
+        }
+        return scheme + "://" + host + path;
+    }
+
+    @Nullable
```

Comment:
can we add a todo for dropping the backward compatibility till 4.0 java agent update or exporter GA?

Reply:
added comment at top of class